ruby/automated_testing/rspec_part_two_code_sharing.md

```diff
@@ -41,7 +41,7 @@ end
 
 Next, create a test file for the user class in the spec directory named `user_spec.rb` and write the following tests for the `User` class to exercise the name, email and age attributes:
 
-~~~ruby
+~~~~ruby
```

Comment:
Code fences need to be wrapped with 3 tildes, not 4. This seems like an incorrect change.